acceptance/README.md

```diff
@@ -0,0 +1,208 @@
+BOLT_CONTROLLER=centos7-64                  \
+BOLT_NODES=ubuntu1604-64,windows10ent-64    \
+SSH_PASSWORD='S3@ret3'                      \
+WINRM_PASSWORD='S3@ret3'                    \
+bundle exec rake ci:test:git
+```
+
+### Beaker
+The above rake tasks are the simplest method of executing the acceptance tests
+for `bolt`. However, they assume that the default output of
+`beaker-hostgenerator` will meet your needs. If this is not the case, you can
+use `beaker-hostgenerator` to create the template for your beaker hosts file,
+modify it to suite your infrastructure, and run beaker independently. This can
+be helpful if you have existing hosts or do not want to use the `vmpooler`
+hypervisor.
+
+####  Hosts file
+
+You should run `beaker-hostgenerator` to create the hosts.yaml file used by
+beaker to run the tests. The examples below illustrate common platforms to
+test against.
+_Centos 7_
+The following example generates a host file with a Centos 7 bolt controller
+with other operating systems used as external nodes to be acted upon by
+`bolt`.
+```
+ bundle exec beaker-hostgenerator centos7-64bolt,ssh.-ubuntu1604-64ssh.-osx1012-64ssh.-windows10ent-64winrm. > hosts.yaml
+```
+_Ubuntu 16.04_
+The following example generates a host file with a Ubuntu 16.04 bolt controller
+with other operating systems used as external nodes to be acted upon by
+`bolt`.
+```
+ bundle exec beaker-hostgenerator ubuntu1604-64bolt,ssh.-centos7-64ssh.-osx1012-64ssh.-windows10ent-64winrm. > hosts.yaml
+```
+_Windows 10 Enterprise_
+The following example generates a host file with a Windows 10 bolt controller
+with other operating systems used as external nodes to be acted upon by
+`bolt`.
+```
+ bundle exec beaker-hostgenerator windows10ent-64bolt,winrm.-centos7-64ssh.-ubuntu1604-64ssh.-osx1012-64ssh. > hosts.yaml
+```
+_OS X 10.12_
+The following example generates a host file with a OS X bolt controller
+with other operating systems used as external nodes to be acted upon by
+`bolt`.
+```
+  bundle exec beaker-hostgenerator osx1012-64bolt,ssh.-centos7-64ssh.-ubuntu1604-64ssh.-windows10ent-64winrm. > hosts.yaml
+```
+
+#### Run test suite with beaker
+The `BOLT_CONTROLLER` and `BOLT_NODES` values are not needed when running
+beaker directly. They are used by `beaker-hostgenerator` to create the hosts
+file for the rake tasks. Since the rake tasks set the defaults for most of the
+other environment variables, you will need to explicitly set them when running
+beaker directly. The beaker options file will also need to be specified for
+the type of bolt installation that should be performed (gem or git).
+
+Example of running test suite using gem install using an existing hosts file:
+```
+SSH_USER=root                      \
+SSH_PASSWORD='S3@ret3'             \
+WINRM_USER=Administator            \
+WINRM_PASSWORD='S3@ret3'           \
+  bundle exec beaker               \
+      -o config/gem/options.rb     \
+      -h hosts.yaml                \
+      -t tests
```

Comment:
Can we use beaker's subcommands, e.g. `beaker init`, `beaker provision`, etc?

Reply:
Yes, the downside is that unless the user wanted to export the password variables they would need to pass them to the pre-suite call and the test run. So, the above command would transform into:

1. Initialize beaker with supplied options file for gem configuration.
```
bundle exec beaker init -h hosts.yaml -o config/gem/options.rb
```
2. Provision hosts
```
bundle exec beaker provision
```
3. Run pre-suite
```
SSH_USER=root                      \
SSH_PASSWORD='S3@ret3'             \
WINRM_USER=Administator            \
WINRM_PASSWORD='S3@ret3'           \
  bundle exec beaker exec pre-suite
```
4. Run tests with specified environment variables
```
SSH_USER=root                      \
SSH_PASSWORD='S3@ret3'             \
WINRM_USER=Administator            \
WINRM_PASSWORD='S3@ret3'           \
  bundle exec beaker exec ./tests
```
5. Destroy hosts
```
bundle exec beaker destroy
```